website/docs/r/orchestrated_virtual_machine_scale_set.html.markdown

```diff
@@ -51,6 +51,8 @@ resource "azurerm_orchestrated_virtual_machine_scale_set" "example" {
 
 * `additional_capabilities` - (Optional) An `additional_capabilities` block as defined below.
 
+* `additional_unattend_content` - (Optional) One or more `additional_unattend_content` blocks as defined below. Changing this forces a new resource to be created.
```

Comment:
This will need to be moved into the right part of the documentation (under the `windows_configuration` block)